### PR TITLE
Add categories

### PR DESCRIPTION
**Pull request by djmitche**
image

This also updates the stability of auth.sentryDSN since we are now supporting that endpoint again :)

### CODE REVIEW

File: changelog/WGyqIUL3Se6bP0_yWYR6Jw.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+level: silent
```
Comment (owlishDeveloper) [inline changelog/WGyqIUL3Se6bP0_yWYR6Jw.md]: why silent if the difference is in the UI?

File: clients/client-py/taskcluster/generated/aio/auth.py [hunk 2]
```diff
@@ -447,7 +447,7 @@ class Auth(AsyncBaseClient):
         initial team configured for this component. Contact a Sentry admin
         to have the project transferred to a team you have access to if needed
 
-        This method is ``deprecated``
+        This method is ``stable``
```
Comment (owlishDeveloper) [inline clients/client-py/taskcluster/generated/aio/auth.py]: why do we resurrect it?

File: changelog/WGyqIUL3Se6bP0_yWYR6Jw.md (hunk 1 above)
Comment (djmitche) [reply changelog/WGyqIUL3Se6bP0_yWYR6Jw.md]: From https://github.com/taskcluster/taskcluster/blob/master/dev-docs/best-practices/changelog.md#levels
> * silent: Changes to tests or documentation
> * silent: Minor changes or fixes to UI

File: clients/client-py/taskcluster/generated/aio/auth.py (hunk 2 above)
Comment (djmitche) [reply clients/client-py/taskcluster/generated/aio/auth.py]: At one point we were going to stop supporting sentry within the platform, when we were using stackdriver for error reporting.  But we reversed course and now report errors directly to sentry.  And, workers use this method to get sentry credentials to report their own errors.  So basically we are no longer planning to remove this method.

File: changelog/WGyqIUL3Se6bP0_yWYR6Jw.md (hunk 1 above)
Comment (owlishDeveloper) [reply changelog/WGyqIUL3Se6bP0_yWYR6Jw.md]: I wonder if I'll ever get used to that scale.... somehow everything is 2 tones down in there comparing to how I'd personally classify changes 😄

**Pull request merged**